canvas_modules/harness/.nycrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Can you explain why this is needed?

Reply:
.nycrc file is needed to configure NYC which generates code coverage reports.

Reply:
Can you give more details on this?  For example how does webpack get used and is there any documentation on this?

Reply:
I removed this file and it doesn't seem to make a different in the result.